test/testtls/generate.go

```diff
@@ -1,21 +1,24 @@
 package testtls
 
 import (
+	"bytes"
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
 	"fmt"
-	stdlog "log"
+	"math/big"
+	"net"
 	"os"
 	"path/filepath"
 	"time"
-
-	"github.com/cloudflare/cfssl/cli/genkey"
-	"github.com/cloudflare/cfssl/config"
-	"github.com/cloudflare/cfssl/csr"
-	"github.com/cloudflare/cfssl/helpers"
```

Comment:
I needed to refactor this file to get rid of this dependency to allow the deeps bump. 


```
github.com/observatorium/api/test/testtls imports
        github.com/cloudflare/cfssl/helpers imports
        github.com/google/certificate-transparency-go imports
        go.etcd.io/etcd/v3 imports
        go.etcd.io/etcd/tests/v3/integration imports
        go.etcd.io/etcd/pkg/v3/grpc_testing imports
        google.golang.org/grpc/test/grpc_testing: module google.golang.org/grpc@latest found (v1.58.3), but does not contain package google.golang.org/grpc/test/grpc_testing
make: *** [Makefile:94: deps] Error 1
```


https://pkg.go.dev/google.golang.org/grpc/test/grpc_testing is published only until 1.54.1